web/src/locales/zh/data.json

```diff
@@ -154,7 +154,7 @@
   },
   "forget": {
     "Account": "账号",
-    "Change Password": "修改密码",
+    "Password change required": "修改密码",
```

Comment:
use the existing `user:Password change required`, it's the same word

Reply:
fixed